index.html

```diff
@@ -16,14 +18,20 @@
         <header>
             <div class ="container-nav">
                 <a href="#" class="logo"> Logo </a>
-                <nav>
-                <a href="#leave_a_message"><strong> Leave a Message</strong></a>
-                <a href="#connect"><strong> Connect </strong></a>
-                <a href="#projects"><strong> Projects </strong></a>
-                <a href="#skills"><strong> Skills </strong></a>
+
+                <nav id="myNav">                          
+                <a class="active" href="#home"><strong> Home </strong></a>
+                <a href="#about"><strong> About </strong></a> 
                 <a href="#experience"><strong> Experience </strong></a>
-                <a href="#about"><strong> About </strong></a>
-                <a class="active" href="#home"><strong> Home </strong></a> 
+                <a href="#skills"><strong> Skills </strong></a>
+                <a href="#projects"><strong> Projects </strong></a>
+                <a href="#connect"><strong> Connect </strong></a>
+                <a href="#leave_a_message"><strong> Leave a Message</strong></a>
+                
+                <!--to show a hamburger menu (bars) on small screens -->
+                <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                    <i class="fa fa-bars"></i>
+                    </a>
```

Comment:
Great idea to use a more compact navigation style for smaller resolution screens. Maybe it's not fully implemented yet, but I am see the hamburger menu when I simulate a smaller screen.